terraform/spec/dependabot/terraform/file_parser_spec.rb

```diff
@@ -939,7 +939,7 @@
       before do
         artifactory_repo_url = "http://artifactory.dependabot.com/artifactory/tf-modules/azurerm"
 
-        stub_request(:get, "#{artifactory_repo_url}/terraform-azurerm-nsg-rules.v1.1.0.tar.gz?terraform-get=1")
```

Comment:
This test was failing because the .tar.gz extension now returns as raw_source rather than following the terraform-get=1 query param flow. I updated the url so the test would reach the desired part of the method under test.